app/org/sagebionetworks/bridge/hibernate/HibernateAccount.java

```diff
@@ -47,6 +47,23 @@
     private Set<Roles> roles;
     private AccountStatus status;
     private int version;
+    private String clientData;
+
+    public HibernateAccount() {}
+    
+    /**
+     * Constructor to load information for the AccountSummary object.
+     */
+    public HibernateAccount(Long createdOn, String studyId, String firstName, String lastName, String email, String id,
+            AccountStatus status) {
```

Comment:
This constructor appears unused except for unit tests. Should we even have this constructor? How do we decide what fields should and shouldn't be included in the constructor?

I think it would be better off to use the default no-arg constructor and just use getters and setters. Otherwise, we end up in a situation where we need to update N^2 places every time we update the constructor.

Reply:
Added comment about the fact that it seems required for Hibernate to construct the object using only the fields specified in the select clause (avoiding joins and the potentially large clientData object).